build/unified/rhel.Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
+FROM registry.access.redhat.com/rhel8/go-toolset:1.11.13-18 as builder
```

Comment:
$ docker pull registry.access.redhat.com/rhel8/go-toolset:1.11.13-18
Error response from daemon: unsupported: This repo requires terms acceptance and is only available on registry.redhat.io

AFAIK it can't be upstream then

Reply:
well, we can switch it to reg.rh.io ... or go back to using rhel7 go toolset to build. which would you prefer?